src/AutoRest.CSharp/LowLevel/Generation/LowLevelClientWriter.cs

```diff
@@ -498,13 +498,21 @@ private static CodeWriter.CodeWriterScope WriteClientMethodDeclaration(CodeWrite
         {
             var methodSignature = clientMethod.Signature.WithAsync(async);
 
-            writer.WriteMethodDocumentation(methodSignature);
+            WriteMethodDocumentation(writer, methodSignature);
             WriteSchemaDocumentationRemarks(writer, clientMethod.OperationSchemas);
             var scope = writer.WriteMethodDeclaration(methodSignature);
             writer.WriteParametersValidation(methodSignature.Parameters);
             return scope;
         }
 
+        private static void WriteMethodDocumentation(CodeWriter codeWriter, MethodSignature methodSignature)
```

Comment:
Do we need to add a new function? Can we update writer.WriteMethodDocumentation function to include the additional part? writer.WriteMethodDocumentation is to write basic document for a method, all the content of basic document will be implemented in that function.

Reply:
`CodeWriter` is commonly shared by mgmt, HLC, andn LLC. The change here is only for LLC, we'd better keep it off the `CodeWriter`.

Reply:
I'm afraid not. The original method is used in a lot of places. I believe this change is only for DPG.